apps/cyberstorm-nextjs/app/communities/CommunityList.tsx

```diff
@@ -5,9 +5,14 @@ import { useDapper } from "@thunderstore/dapper";
 import { usePromise } from "@thunderstore/use-promise";
 
 import styles from "./CommunityList.module.css";
-import { SortOptions } from "./CommunityListLayout";
-import { CommunityCard } from "../../CommunityCard/CommunityCard";
-import * as EmptyState from "../../EmptyState";
+import { CommunityCard } from "@thunderstore/cyberstorm/src/components/CommunityCard/CommunityCard";
+import { EmptyState } from "@thunderstore/cyberstorm";
+
+enum SortOptions {
```

Comment:
Why duplicate the options here?

Reply:
It wouldn't work with imports, because of NextJS. NextJS somehow can't handle imports between pages and layouts. Still thinking about a solution for this. Might just make a separate file for constants in the nextjs app